lib/wkhtmltopdf_binary.rb

```diff
@@ -1,15 +1,14 @@
 class WkhtmltopdfBinary
   def self.path
     @path ||= begin
-      arch = case RUBY_PLATFORM
-        when /linux/
-          'linux-x86'
-        when /darwin/
-          'darwin-x86'
-        else
-          raise "Invalid platform. Must be running linux or intel-based Mac OS."
+      cpu, os = RUBY_PLATFORM.split('-', 2)
+      if os != 'linux' && os != 'darwin'
+        raise "Invalid platform. Must be running linux or intel-based Mac OS."
       end
-      File.expand_path "#{File.dirname(__FILE__)}/../libexec/wkhtmltopdf-#{arch}"
+      path = "#{File.dirname(__FILE__)}/../libexec/"
```

Comment:
I prefer to use `File.join` to string concatenation for file paths - it's very easy to make mistake in the string concats.